enzyme/Enzyme/MLIR/Dialect/EnzymeOps.td

```diff
@@ -79,8 +79,7 @@ def PlaceholderOp : Enzyme_Op<"placeholder",
 def ForwardDiffOp : Enzyme_Op<"fwddiff",
     [DeclareOpInterfaceMethods<SymbolUserOpInterface>]> {
   let summary = "Perform forward mode AD on a funcop";
-  let arguments = (ins FlatSymbolRefAttr:$fn, Variadic<AnyType>:$inputs, ActivityArrayAttr:$activity, ActivityArrayAttr:$ret_activity);
-  let results = (outs Variadic<AnyType>:$outputs);
+  let arguments = (ins FlatSymbolRefAttr:$fn, Variadic<AnyType>:$inputs, ActivityArrayAttr:$activity, ActivityArrayAttr:$ret_activity, DefaultValuedAttr<I64Attr, "1">:$width);  let results = (outs Variadic<AnyType>:$outputs);
```

Comment:
I didn't define a custom Enzyme Attribute because I didn't find how to set a default value easily for it.

Reply:
honestly an int attribute is appropriate here anyways